src/bootstrap/check.rs

```diff
@@ -110,12 +110,11 @@ fn testdir(build: &Build, host: &str) -> PathBuf {
 /// Compiles all tests with `compiler` for `target` with the specified
 /// compiletest `mode` and `suite` arguments. For example `mode` can be
 /// "run-pass" or `suite` can be something like `debuginfo`.
-pub fn compiletest(build: &Build,
-                   compiler: &Compiler,
-                   target: &str,
-                   mode: &str,
-                   suite: &str) {
-    println!("Check compiletest {} ({} -> {})", suite, compiler.host, target);
+pub fn compiletest(build: &Build, compiler: &Compiler, target: &str, mode: &str, suite: &str) {
+    println!("Check compiletest {} ({} -> {})",
+             suite,
+             compiler.host,
+             target);
```

Comment:
There's a heuristic for breaking function calls (and macro uses) before they fill the whole line.